docs/pages/reference/cli.mdx

```diff
@@ -159,10 +159,42 @@ $ tsh help
 
 ### tsh version
 
-Print the version of your `tsh` binary:
+Print the version of your `tsh` binary and current profile Teleport proxy version:
```

Comment:
"Current profile proxy version" sounds a bit odd to me. How about something like "the version of the Teleport proxy pointed to by your current tsh profile."

Reply:
addressed